### DISCUSSION TITLE
Color of the read more button in the slideshow widget

### DISCUSSION THREAD

**Discussion by Banuk1616** [templaza/astroid-framework#744] [Q&A]
Maybe someone can help me out? I would like to change the color of the read more button in the slideshow widget to a color that harmonizes better with the color of the overlay.
The button has the class btn btn-primary by default and I'm trying to write my own class for the button but I'm not getting anywhere at the moment. Can anyone give me a tip? Many thanks in advance and best regards.

**Comment by Chacapamac** [comment]
Maybe yo can add the container div of the “btn btn-primary” class like
div#whateveristhecontainername btn.btn-primary
or
div.whateveristhecontaineclass btn.btn-primary
and specify the color. if not working, add the code “ !important” at the en of the color specification.

**Reply by Banuk1616** [reply]
@Chacapamac

Thanks for the tip. I'm a bit short of time again at the moment, but I'll try it out next week.

**Comment by sonvnn** [comment]
At the moment, You can use Custom css to do. I will consider to add an option to allow admin define button style in the next version.

**Comment by Banuk1616** [comment]
@sonvnn

I was aware that it only works with custom.css, but I didn't know how to realise it with css. I'm not very good at it yet.

I think it would be great if you could make it possible to define the colour of the button in the next version. I'm already looking forward to it.

It would also be great if you could define the colour of the overlay-inner in the widget without custom.css

**Comment by Chacapamac** [comment]
>  if you could define the colour of the overlay-inner in the widget without custom.css
Yep, it will be cool!
